src/utils.py

```diff
@@ -0,0 +1,102 @@
+import json
+
+
+def show_menu(options: dict[int,str]) -> int:
+    """
+    Kiírja a menüt a megadott opciókkal majd visszaadja a felhasználó választását.
+
+    Args:
+    options(dict[int, str]): Menü opciók, ahol a kulcsok a menüpont sorszámai, az értékek pedig a nevek.
+
+    Returns:
+    int: A felhasználó által választott menüpont sorszáma.
+    """
+    
+    while True:
+        print("--------------------")
+        for number,title in options.items():
+            print(f"{number} - {title}")
+        print("--------------------")
+        try:
+            choice = int(input("Válasszon menüpontot: "))
+        except ValueError:
+            print("HIBA! Csak számot adhat meg!")
+            print()
+            continue
+            
+        if choice in options: return choice
+        else: print("HIBA! Nincs ilyen menüpont!"); print()
```

Comment:
Üres printek helyett `\n`:
```suggestion
            print("HIBA! Csak számot adhat meg!\n")
            continue
            
        if choice in options: return choice
        else: print("HIBA! Nincs ilyen menüpont!\n");
```